lib/sumodev_deploy/tasks/assets.rb

```diff
@@ -19,6 +19,11 @@
       else
         logger.important "No Gruntfile.coffee or gulpfile.js found"
       end
+      if File.exists?("package.json")
+        run_locally "rm -rf temporary_node_modules; mkdir temporary_node_modules; cp package.json temporary_node_modules; cd temporary_node_modules; npm install --production"
+        upload "./temporary_node_modules", "#{latest_release.shellescape}"
```

Comment:
Have you tried to run it, does it upload properly? Since it's a directory, I think you may need to add `:recursive => true` as a third parameter.
Here are the docs for Capistrano 2 (I think this gem uses cap 2, since Capistrano 3 uses `upload!` command, with a bang at the end):
https://github.com/capistrano/capistrano-2.x-docs/blob/master/2.x-DSL-Action-File-Transfer-Upload.md